src/excmds.ts

```diff
@@ -833,6 +833,32 @@ async function getnexttabs(tabid: number, n?: number) {
 //#background_helper
 import * as controller from './controller'
 
+/** Repeats a `cmd` `n` times.
+    Falls back to the last executed command if `cmd` doesn't exist.
+    Executes the command once if `n` isn't defined either.
+*/
+//#background
+export function repeat(...args: string[]) {
+    let n = 1
+    let cmd = getLastExstr()
+    if (args.length > 0)
+        n = parseInt(args[0])
+    if (isNaN(n)) {
+        console.log(":repeat error: " + args[0] + " is not a number")
+        return
+    }
+    if (args.length > 1)
+        cmd = args.slice(1).join(" ")
+    console.log("repeating " + cmd + " " + n + " times")
+    for (let i = 0; i < n; ++i)
```

Comment:
Pre-increment is rarer and may confuse novices. Consider using `for of range()` from itertools or using post increment.